tck/pom.xml

```diff
@@ -286,6 +286,14 @@
         </dependency>
     </dependencies>
 
+    <repositories>
```

Comment:
I'd assume it is inherited from parent already, but needs `staging` profile activation.